includes/models/model.llms.student.php

```diff
@@ -1841,4 +1909,171 @@ private function update_completion_status( $status, $object_id, $object_type, $t
 
 	}
 
+	/**
+	 * Determine if the student has favorited a lesson.
+	 *
+	 * @since [version]
+	 *
+	 * @param    int    $object_id      WP Post ID of a course or lesson or section or the term id of the track.
+	 * @param    string $object_type    Object type (course, lesson, section, or track).
+	 * @return   boolean
```

Comment:
can you fix the docblock with all already said?

Reply:
fixed.